src/extra/widgets/span/lv_span.c

```diff
@@ -434,10 +434,13 @@ lv_coord_t lv_spangroup_get_max_line_h(lv_obj_t * obj)
 }
 
 /**
- * get the width when all span of spangroup on a line. not included spangroup pad, border width.
+ * get the text content width when all span of spangroup on a line.
```

Comment:
Now both comments say:
> get the text content width when all span of spangroup on a line.


BTW, in other files we added API comments only in the header files to avoid duplication. Please follow this pattern with the span widget too?